1/right_triangle.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+# Program requests three sides of triangle and defines
+# whether triangle rectangular. Also display notification
+# if triangle is isosceles or equilateral.
+# Calculation carried using the Pythagorean theorem
+
+TRIANGLE_TYPE_MESSAGES = {
+  equilateral_triangle: 'Triangle isosceles and equilateral',
+  isosceles_triangle: 'Isosceles triangle',
+  rectangle_triangle: 'The triangle is rectangular',
+  not_right_triangle: 'The triangle is not rectangular'
+}.freeze
+
+def pythagorean_theorem(sides)
```

Comment:
1. Don't use short and not understandable variables names
2. Rebuild this method in at most two lines.